docs/providers/aws/events/activemq.md

```diff
@@ -54,9 +54,10 @@ functions:
           basicAuthArn: arn:aws:secretsmanager:us-east-1:01234567890:secret:MySecret
 ```
 
-## Specifying batch size
+## Specifying batch size and batch window
```

Comment:
let's reference that property in `serverless.yml.md` file

Reply:
Updated in 9f418fc917785ca522dd20f1c61169d3d8eee3ed
Note: I added it at the same position in `serverless.yaml.md` as #10591, so likely to be a merge conflict (both need to be accepted) ... i can fix that as soon as the other is merged though